pkg/image/api/types.go

```diff
@@ -157,3 +195,42 @@ type DockerImageReference struct {
 	Tag       string
 	ID        string
 }
+
+type ImageStreamImport struct {
```

Comment:
The flag is new, making it consistent with other usage.

On Mon, Jan 11, 2016 at 12:18 PM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> This is post 1.1.1
>
> On Mon, Jan 11, 2016 at 10:31 AM, David Eads <notifications@github.com>
> wrote:
>
>> In pkg/image/api/types.go
>> <https://github.com/openshift/origin/pull/6288#discussion_r49333882>:
>>
>> > @@ -157,3 +195,42 @@ type DockerImageReference struct {
>> >  	Tag       string
>> >  	ID        string
>> >  }
>> > +
>> > +type ImageStreamImport struct {
>>
>> Need GoDoc on these.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openshift/origin/pull/6288/files#r49333882>.
>>
>
>